### ISSUE TITLE
Icon or thumbnail for image file

### ISSUE THREAD

**Issue by dixours** [ffMathy/Shapeshifter#58]
Hi !
When I copy image files, is there a way to see thumbnails in the list ? Or, maybe, just an icon which looks like an image ?

**Comment by ffMathy** [comment]
Yes, this will be implemented in https://github.com/ffMathy/Shapeshifter/issues/21.

I will keep this issue open to track progress though.
